Renderer performance

degenerated1123 opened the pull request:
Implemented frustum-culling for most objects

Performance on Android has been significantly improved. On Desktop, it hardly dips below 300fps now in the old camp for me (before ~80).

markusobi commented:
The performance improvement is great, but there is an issue with NPCs (i.e. sheeps) popping up too late at the screen edges.

![sheep](https://user-images.githubusercontent.com/16803864/31678130-6dc0581a-b36d-11e7-81ee-86ec20801ec6.png)
![nosheep](https://user-images.githubusercontent.com/16803864/31678136-7338e0dc-b36d-11e7-885f-92167621cfb2.png)

degenerated1123 commented:
Fixed the issue with the sheeps: They actually didn't have a bounding-box assigned the same way as the other skeletal meshes.
Also fixed a similar issue with the particle effects popping out of view too early.

If no other issues are found, this would be ready to be merged again!
